contents/newsletter/how-to-get-job-startup.md

```diff
@@ -0,0 +1,111 @@
+---
+title: How to get a great job at a great startup
+date: 2024-07-18
+author:
+  - james-temperton
+featuredImage: >-
+  https://res.cloudinary.com/dmukukwp6/image/upload/v1713521535/posthog.com/contents/blog/evolution-of-founders.jpg
+featuredImageType: full
+tags:
+  - People
+crosspost:
+  - Blog
+---
+
+We get a lot of job applications. In the last two months, we’ve had 900 people apply for engineering roles:
+
+* 86 got a first interview
+* 55 got a technical interview
+* 10 did a [SuperDay](https://posthog.com/handbook/people/hiring-process#4-posthog-superday) 
+* 4 were hired 
+
+![hiring engineers](https://res.cloudinary.com/dmukukwp6/image/upload/hiring_hogs_e172cf51da.png)
+
+That means we hired 0.4% of people who applied for engineering roles.
+
+So, how do you join the 0.4% of superstar engineers – not just at PostHog, but at any other fast-growing startup?
+
```

Comment:
Unwritten step 1, be good. How to prove this:
- Other people saying you are good (AKA referrals) are the best way to do this.
- Tangible proof you've done the thing the company needs (build products 0-1, scale infrastructure, improve querying, etc)

Reply:
this is a very good point. let me find the right place for it and work it in